kumascript/macros/Obsolete_Inline.ejs

```diff
@@ -4,38 +4,28 @@
 // NOTE: "Obsolete" means that the indicated item NO LONGER FUNCTIONS. If
 //       the item still works, but should no longer be used, it's "deprecated".
 //
-// Parameters:
-//
-//  $0  Version of obsolescence
```

Comment:
This parameter is still used by 111 of 360 occurrences. I don't think we should silently drop it.

Looking at `git log -S 'obsolete_inline' -p`, it seems that replacing any `{{obsolete_inline}}` with `{{deprecated_inline}}` paves the way to removing this macro.

@queengooborg Would it make sense to open two PRs that replace the remaining usages in `pt-BR` and `ru` respectively? And then we could remove these three obsolete macros altogether?